src/Shared/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,112 @@
+.sidebar{
+    background-color: #ee9da5;
+    position: fixed;
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+    width: 15rem;
+    height: 100vh;
+    position: fixed;
+    top: 4rem;
+    left: 0;
+    height: 100%;
+    padding: 14px 0;
+    height:98%;
+}
+
+.logo_content{
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo{
+    width: 100%;
+}
+
+.logo img{
+    border-radius: 15%;
+}
+
+.nav_list{
+    display: flex;
+    flex-direction: column;
+}
+
+.nav_list li:last-child{
+    background-color: transparent;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    width: 15rem;
+}
+
+
+
+.sidebar .nav_list a{
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    padding: 10px;
+    transition: all 0.4s ease;
+}
+
+.sidebar .nav_list .nav-icon{
+   margin:0 10px;
+}
+
+.sidebar #btn{
+    position: absolute;
+    left: 90%;
+    top: 6px;
+    transform: translateX(-50%);
+}
+
+.profile_content{
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    position: absolute;
+    bottom: 5rem;
+    padding: 10px;
+    color: var(--white-color);
+    font-weight: 500;
+}
+
+.profile_content .profile{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap:10px;
+}
+
+.profile_content #log-out{
+    position: absolute;
+    margin: 0;
+    margin-right: 10px;
+    right: 0%;
+    transform: translateX(-50%);
+}
+
+
+
+@media (max-width:768px){
+    .sidebar{
+        width: 100%;
+        z-index: 98;
```

Comment:
I saw z-index: 98 multiple times in codebase and feel like it can become tough to debug code in future.

Reply:
this inex is for alert or modal